backend/Dockerfile

```diff
@@ -10,12 +10,14 @@ RUN apk add --update jq curl py-pip inotify-tools
 RUN pip install --no-cache-dir -r requirements.txt
 
 # Copy the Python files and .env
-COPY nova_s2s_backend.py knowledge_base_lookup.py retrieve_user_profile.py cognito.py entrypoint.sh .env ./
+COPY nova_s2s_backend.py knowledge_base_lookup.py retrieve_user_profile.py cognito.py entrypoint.sh tools.py mcp_server.py .env ./
```

Comment:
Let's do `COPY . ./` (copy everything in the folder) instead? We have `.dockerignore` so that we don't have to list the files. Otherwise we'd need to remind people to add their tool files here.